docs/standard/design-guidelines/choosing-between-anonymous-and-tuple.md

```diff
@@ -83,23 +83,30 @@ foreach (var (formatted, ticks) in
 
-While it might seem as though you should always use <xref:System.ValueTuple> over <xref:System.Tuple>, and anonymous types - there are tradeoffs you should consider. The <xref:System.ValueTuple> types are mutable, whereas <xref:System.Tuple> are read-only. Anonymous types can be used in expression trees, while tuples cannot.
+The previous examples are all functionally equivalent, however; there are slight differences in their usability and their underlying implementations.
+
+## Tradeoffs
+
+While it might seem as though you should always use <xref:System.ValueTuple> over <xref:System.Tuple>, and anonymous types - there are tradeoffs you should consider. The <xref:System.ValueTuple> types are mutable, whereas <xref:System.Tuple> are read-only. Anonymous types can be used in expression trees, while tuples cannot. The following table is an overview of some of the key differences.
 
 ### Key differences
 
-| Name                     | Type     | Custom property name | Deconstruction support | Expression tree support |
-|--------------------------|----------|----------------------|------------------------|-------------------------|
-| Anonymous types          | `class`  | ✔️                  | ❌                      | ✔️                     |
-| <xref:System.Tuple>      | `class`  | ❌                   | ❌                      | ✔️                     |
-| <xref:System.ValueTuple> | `struct` | ✔️                  | ✔️                     | ❌                      |
+| Name                     | Access modifier | Type     | Custom property name | Deconstruction support | Expression tree support |
+|--------------------------|-----------------|----------|----------------------|------------------------|-------------------------|
+| Anonymous types          | `internal`      | `class`  | ✔️                   | ❌                     | ✔️                     |
+| <xref:System.Tuple>      | `public`        | `class`  | ❌                   | ❌                     | ✔️                     |
+| <xref:System.ValueTuple> | `public`        | `struct` | ✔️                   | ✔️                     | ❌                     |
 
 ## Performance
 
-Performance is always a topic of discussion when referring to value types vs reference types, often leading to conversations of stack versus heap allocations. However, performance between these types is fairly similar with each other - in other words, performance is irrelevant. The micro-optimizations you might gain from choosing tuples over anonymous types is more often than not negligible.
+Performance is always a topic of discussion when referring to value types vs reference types, often leading to conversations of stack versus heap allocations. However, performance between these types is fairly similar with each other - in other words, performance differences are negligible. The micro-optimizations you might gain from choosing tuples over anonymous types is more often than not irrelevant.
```

Comment:
I'll suggest wording similar to the following. The performance differences *could* be major, but it depends on how either type is used:

```suggestion
Performance between these types depends on the scenario. The major impact involves the tradeoff between allocations and copying. In most scenarios, the impact is small. When major impacts could arise, measurements should be takent to inform the decision.
```